app/helpers/test_helpers.go

```diff
@@ -161,11 +140,7 @@ func setup(withGenesis bool, invCheckPeriod uint) (*gaiaapp.GaiaApp, gaiaapp.Gen
 		encCdc,
 		EmptyAppOptions{},
 	)
-	if withGenesis {
-		return gaiaApp, gaiaapp.NewDefaultGenesisState()
-	}
-
-	return gaiaApp, gaiaapp.GenesisState{}
+	return gaiaApp, gaiaapp.NewDefaultGenesisState()
```

Comment:
We use NewDefaultGenesisState, so MakeTestEncodingConfig is setup ... what does this give us?